packages/design-system-tokens/sketch/token-importer/README.md

```diff
@@ -0,0 +1,92 @@
+# token-importer
+
+## Installation
+
+- [Download](../../releases/latest/download/token-importer.sketchplugin.zip) the latest release of the plugin
+- Un-zip
+- Double-click on token-importer.sketchplugin
+
+## Development Guide
+
+_This plugin was created using `skpm`. For a detailed explanation on how things work, checkout the [skpm Readme](https://github.com/skpm/skpm/blob/master/README.md)._
+
+### Usage
+
+Install the dependencies
+
+```bash
+npm install
+```
+
+Once the installation is done, you can run some commands inside the project folder:
+
+```bash
+npm run build
+```
+
+To watch for changes:
+
+```bash
+npm run watch
+```
+
+Additionally, if you wish to run the plugin every time it is built:
+
+```bash
+npm run start
+```
+
+### Custom Configuration
+
+#### Babel
+
+To customize Babel, you have two options:
+
+- You may create a [`.babelrc`](https://babeljs.io/docs/usage/babelrc) file in your project's root directory. Any settings you define here will overwrite matching config-keys within skpm preset. For example, if you pass a "presets" object, it will replace & reset all Babel presets that skpm defaults to.
+
+- If you'd like to modify or add to the existing Babel config, you must use a `webpack.skpm.config.js` file. Visit the [Webpack](#webpack) section for more info.
+
+#### Webpack
+
+To customize webpack create `webpack.skpm.config.js` file which exports function that will change webpack's config.
+
+```js
+/**
+ * Function that mutates original webpack config.
+ * Supports asynchronous changes when promise is returned.
+ *
+ * @param {object} config - original webpack config.
+ * @param {boolean} isPluginCommand - whether the config is for a plugin command or a resource
+ **/
+module.exports = function (config, isPluginCommand) {
+  /** you can change config here **/
+};
+```
```

Comment:
[questions] are there custom webpack or babel configs for this directory that merits this documentation? Or is this proactively trying to answer future questions?

Reply:
Yeah this is future cleanup stuff, this whole file will change once the plugin is done (other type imports)